src/server/authserver/Main.cpp

```diff
@@ -113,7 +113,25 @@ extern int main(int argc, char** argv)
 
     sLog->outString("%s (authserver)", _FULLVERSION);
     sLog->outString("<Ctrl-C> to stop.\n");
-    sLog->outString("Using configuration file %s.", configFile);
```

Comment:
you're right, the problem is that you changed the code indentation... can you fix it (in all new lines)?